pkg/runtime/sched_time.go

```diff
@@ -0,0 +1,76 @@
+package runtime
+
+import (
+	"encoding/json"
+	"bytes"
+	"fmt"
+)
+/*
+SUBMIT_JOB_SCHED
+
+    Just like SUBMIT_JOB_BG, but run job at given time instead of
+    immediately.
+
+    Arguments:
+    - NULL byte terminated function name.
+    - NULL byte terminated unique ID.
+    - NULL byte terminated minute (0-59).
+    - NULL byte terminated hour (0-23).
+    - NULL byte terminated day of month (1-31).
+    - NULL byte terminated month (1-12).
+    - NULL byte terminated day of week (0-6, 0 = Monday).
+    - Opaque data that is given to the function as an argument.
+ */
+
+type SpecScheduleTime struct {
+	Minute string
+	Hour   string
+	Dom    string
```

Comment:
Do we need our own object? can we reuse or Embed https://github.com/robfig/cron/blob/v2/parser.go#L53 ?